[2022-06-10 17:01:29] [INFO] [Recovery] 2022/06/10 - 17:01:29 panic recovered:
can't preload field cluster for model.Instance
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/errorx/errorx.go:40 (0x17babfa)
	Dangerous: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/ginx/errorx.go:10 (0x17efeca)
	Dangerous: }
C:/Users/mensyli1/work/workspace/middleware/src/controller/postgres/postgres.go:108 (0x17efeb3)
	Page: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x17f3467)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/jwt.go:19 (0x17f344e)
	Auth.func1: 
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x17f358b)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/logrus.go:62 (0x17f3568)
	Logrus.func1: //ÁªìÊùüÊó∂Èó¥
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x17f24fa)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/recovery.go:94 (0x17f24e6)
	CustomRecoveryWithWriter.func1: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x172ce70)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/gin.go:555 (0x172cad8)
	(*Engine).handleHTTPRequest: c.writermem.WriteHeaderNow()
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/gin.go:511 (0x172c611)
	(*Engine).ServeHTTP: 
C:/install/Dev/Env/Go/src/net/http/server.go:2879 (0xa84a7a)
	serverHandler.ServeHTTP: }
C:/install/Dev/Env/Go/src/net/http/server.go:1930 (0xa80127)
	(*conn).serve: w.cancelCtx()
C:/install/Dev/Env/Go/src/runtime/asm_amd64.s:1581 (0x7d4b00)
	goexit: CALL	runtime¬∑goexit1(SB)	// does not return
[0m (github.com/mensylisir/kmpp-middleware/src/util/aop.CustomRecoveryWithWriter.func1.1: 82) {map[]} 
[2022-06-10 17:01:29] [INFO] [Recovery] 2022/06/10 - 17:01:29 panic recovered:
can't preload field cluster for model.Instance
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/errorx/errorx.go:40 (0x17babfa)
	Dangerous: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/ginx/errorx.go:10 (0x17efeca)
	Dangerous: }
C:/Users/mensyli1/work/workspace/middleware/src/controller/postgres/postgres.go:108 (0x17efeb3)
	Page: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x17f3467)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/jwt.go:19 (0x17f344e)
	Auth.func1: 
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x17f358b)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/logrus.go:62 (0x17f3568)
	Logrus.func1: //ÁªìÊùüÊó∂Èó¥
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x17f24fa)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/recovery.go:94 (0x17f24e6)
	CustomRecoveryWithWriter.func1: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/context.go:168 (0x172ce70)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/gin.go:555 (0x172cad8)
	(*Engine).handleHTTPRequest: c.writermem.WriteHeaderNow()
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.7/gin.go:511 (0x172c611)
	(*Engine).ServeHTTP: 
C:/install/Dev/Env/Go/src/net/http/server.go:2879 (0xa84a7a)
	serverHandler.ServeHTTP: }
C:/install/Dev/Env/Go/src/net/http/server.go:1930 (0xa80127)
	(*conn).serve: w.cancelCtx()
C:/install/Dev/Env/Go/src/runtime/asm_amd64.s:1581 (0x7d4b00)
	goexit: CALL	runtime¬∑goexit1(SB)	// does not return
[0m (github.com/mensylisir/kmpp-middleware/src/util/aop.CustomRecoveryWithWriter.func1.1: 82) {map[]} 
